src/components/WalletTokenList.vue

```diff
@@ -87,6 +93,13 @@ const usdValue = (token: Token) : string => {
   }
   return "0";
 };
+
+watch(() => useTransactionStore().status, async () => {
+  await delay(2000);
```

Comment:
Different chain may have different block confirmation time. 2000ms is not a well-tuned metrics here. Ideally we should refresh once the transaction is finalized on chain. let's just don't refresh for now and get rid of the transaction store.